unit_tests/test_aggregator.py

```diff
@@ -0,0 +1,415 @@
+        agg = aggregator_explicit
+        assert agg.simulation_id == test_simulation["uuid"]
+
+    @staticmethod
+    def test_constructor__simulation_id_from_env(aggregator_from_env):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_from_env.simulation_id == test_simulation["uuid"]
+
+    @staticmethod
+    def test_constructor__domain_name_explicit(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.domain_name == test_simulation["domain_name"]
+
+    @staticmethod
+    def test_constructor__domain_name_from_env(aggregator_from_env):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_from_env.domain_name == test_simulation["domain_name"]
+
+    @staticmethod
+    def test_constructor__websockets_domain_name_explicit(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.websockets_domain_name == \
+               test_simulation["websockets_domain_name"]
+
+    @staticmethod
+    def test_constructor__websockets_domain_name_from_env(aggregator_from_env):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_from_env.websockets_domain_name == \
+               test_simulation["websockets_domain_name"]
+
+    @staticmethod
+    def test_constructor__asset_uuid(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.asset_uuid == ""
+
+    @staticmethod
+    def test_constructor__jwt_token(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.jwt_token == "some_key"
+
+    @staticmethod
+    def test_constructor__aggregator_prefix(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.aggregator_prefix == "/aggregator_prefix/"
+
+    @staticmethod
+    def test_constructor__configuration_prefix(aggregator_explicit):
+        # this attribute is inherited from rest_asset_client
+        assert aggregator_explicit.configuration_prefix == "/configuration_prefix/"
+
+    @staticmethod
+    def test_constructor__grid_fee_calculation():
+        with patch("gsy_e_sdk.aggregator.GridFeeCalculation", autospec=True) as mocked_class:
+            Aggregator(aggregator_name=test_agg["name"])
+            mocked_class.assert_called_once()
+
+    # *
+    @staticmethod
+    def test_constructor__aggregator_name(aggregator_explicit):
+        assert aggregator_explicit.aggregator_name == test_agg["name"]
+
+    # *
+    @staticmethod
+    @pytest.mark.parametrize("accept", [True, False])
+    def test_constructor__accept_all_devices(accept):
+        agg = Aggregator(aggregator_name=test_agg["name"],
+                         accept_all_devices=accept)
+        assert agg.accept_all_devices == accept
+
+    # *
```

Comment:
Please remove these. Thanks!

Reply:
I think @hannesdiedrich meant to just remove the `# *` comments. What are they for?